apps/session-manager/src/services/redis-session-storage.ts

```diff
@@ -601,3 +602,31 @@ export const set =
       TE.map(() => true),
     );
   };
+
+/**
+ * Check if a user is blocked
+ *
+ * @param dependencies - RedisClientSelector and fiscalCode of the user
+ *
+ * @returns a promise with either an error or a boolean indicating if the user is blocked
+ */
+export const isBlockedUser: RTE.ReaderTaskEither<
+  {
+    selector: RedisClientSelectorType;
```

Comment:
use a meaningful name like `redisClientSelector`

Reply:
renamed in https://github.com/pagopa/io-auth-n-identity-domain/pull/22/commits/0aac1a7af5e32060657c60e53ef458e41e4e445c